### DISCUSSION TITLE
Html to wikitext convertor for TW5

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#102] [Ideas]
I was using Mark's HtmltoTW5 convertor to clip from webpages in wikitext. I saw this macro in this google group discussion 

https://groups.google.com/g/tiddlywiki/c/2uh1pQLNKHM/m/CTcPAA8lAgAJ

Here I saw you mention about a html to tw2 converter. Did you ever managed to create a similar convertor for TW5. 

I had a few issues with Mark's convertor which I described in the forum https://talk.tiddlywiki.org/t/help-needed-about-html2tw-macro/4207?u=arunnbabu81

**Comment by buggyj** [comment]
tw2 markup does not include html - so I found conversion useful before tw5. I do not find a need to convert html to tw5 so did not port my tiddlyclip macro to tw5.

**Reply by arunnbabu81** [reply]
Can you look whats is happening with Mark's convertor as I have mentioned in the TW forum. I found it useful when I want to clip from long webpages which has many formatted text and images (in instances I don't want to do use html clipping)
![Screenshot_20220807-095033_Chrome](https://user-images.githubusercontent.com/67494083/183275174-ad3dfe37-cf0d-4eff-a7ff-cfa83145a4a0.jpg)


https://multicolumn-krystal.tiddlyhost.com/

Click on the Tiddlyclip link in the topbar to see the demo

**Reply by buggyj** [reply]
It is extremely difficult to convert html to markup. I would suggest using a markdown addon for firefox that can copy to the clipboard and then use that with tiddlyclip.

**Reply by arunnbabu81** [reply]
I had tried a markdown addon, but it was not working well on the websites I use like GitHub and Radiology websites. Html clipping was better in those sites.

https://github.com/buggyj/tiddlyclip-plugin/discussions/47#discussioncomment-1961270
Can the method shown in the above linked discussion be used in the second issue raised in the forum discussion - https://talk.tiddlywiki.org/t/help-needed-about-html2tw-macro/4207/3?u=arunnbabu81
